Updated docs with local gem instructions

msecret opened the pull request.

Commit:
Updated docs with local gem instructions

afeld commented on README.md:
```diff
@@ -30,8 +30,10 @@ the source files.
+3. Run the watching build task in the *cg-style* repository: `npm run watch`
+4. Replace the `gem 'cloudgov-style'` line in the `Gemfile` with `'cloudgov-style', :github => '18F/cg-style-gem', :branch => 'master'`.
+6. Run `bundle` again.
+7. Run `bundle config local.cloudgov-style ~/path/to/cg-style/repo` wih the path to where you cloned the cg-style repo.
```
Maybe easier to tell them to use the `:path` option?

http://bundler.io/gemfile.html

msecret replied:
So in that case, we'd just tell them to add `, :path => './vendor/extracted_library'` rather then change the whole line? That makes more sense

afeld replied:
Yep, that would allow you to change the "Replace the `gem 'cloudgov-style'`" line, and cut out the `bundle config` step.

Commit:
Use path for local dev of cg-style gem

Use :path for the gem definition instead of :github allows us to
simplify the instructions and skip the `bundle config` step

jeremiak commented:
@afeld this look good now?

afeld commented:
Build failing due to https://github.com/wercker/step-bundle-install/issues/14.
